index.html

```diff
@@ -1,8 +1,37 @@
 <html>
+    <link rel="stylesheet" href="styles.css"> 
     <head>
```

Comment:
Add  a header tag and check this article about the <head> tag https://www.w3schools.com/tags/tag_head.asp